server/data/data-api/src/main/java/org/apache/james/rrt/api/RecipientRewriteTable.java

```diff
@@ -94,7 +94,11 @@ public TooManyMappingException(String string) {
      * @throws ErrorMappingException
      *             get thrown if an error mapping was found
      */
-    Mappings getResolvedMappings(String user, Domain domain) throws ErrorMappingException, RecipientRewriteTableException;
+    default Mappings getResolvedMappings(String user, Domain domain) throws ErrorMappingException, RecipientRewriteTableException {
+            return getResolvedMappings(user, domain, EnumSet.allOf(Mapping.Type.class));
+    }
+
+    Mappings getResolvedMappings(String user, Domain domain, EnumSet<Mapping.Type> mappingTypes) throws ErrorMappingException, RecipientRewriteTableException;
```

Comment:
You should document what the enumset is used for (or make the method name explicit)